Formula/hurl.rb

```diff
@@ -0,0 +1,47 @@
+class Hurl < Formula
+  desc "Run and Test HTTP Requests with plain text and curl"
+  homepage "https://hurl.dev"
+  url "https://github.com/Orange-OpenSource/hurl/archive/refs/tags/1.5.0.tar.gz"
+  sha256 "469cca44022d9339fad246fd417b22703258164fd1cdbe040476a1c127791184"
+  license "Apache-2.0"
+  head "https://github.com/Orange-OpenSource/hurl.git", branch: "master"
+
+  depends_on "rust" => :build
+
+  on_linux do
+    depends_on "curl"
+    depends_on "libxml2"
+    depends_on "openssl@1.1"
+    depends_on "pkg-config" => :build
+    depends_on "python" => :build
+  end
+
+  def install
+    ENV["OPENSSL_DIR"] = Formula["openssl@1.1"].opt_prefix if OS.linux?
+
+    system "cargo", "install", *std_cargo_args(path: "packages/hurl")
+    system "cargo", "install", *std_cargo_args(path: "packages/hurlfmt")
+
+    # We construct manpage with python2 on macOS, and python3 on Linux so
+    # we don't need a brew python dependency on macOS.
+    # We'll remove the python dependency for Linux in the next version.
+    python = if OS.linux?
+      "python3"
+    else
+      "python"
+    end
```

Comment:
We really don't want to use python2 for anything. It's been unsupported for 2 years. If something needs python it can use the system python or the Homebrew python

Reply:
Yes, I completely understand it, that's what I try to do:
- on macOS, the formula uses system python (which is python2), that why I've not added a python dependency on macOS
- on Linux, the formula uses Homebrew python3 (with a brew python dependency)

Reply:
So why can't it use brew python on macOS?

Reply:
On macOs, the dependency seems to me not necessary as the man script generation was working with the system python. 
If it blocked the review on this version, I will close the PR and propose a formula for the next version of Hurl as we are already planning to remove the python dependency on both macOS and Linux (it's only used for man page generation)

Reply:
It should really depend on Homebrew python on build, we can't rely on Apple providing an outdated unsupported python forever and nobody will notice the dependency because:
1. Almost everyone already has something installed that needs python
2. It's only a dependency on build so almost no users will get this dependency pulled in.

Reply:
Ok, that's clear.
I will fix the formula and, in anycase, I will remove the python dependency in the next version

Reply:
@SMillerDev I've rebased and push with the `python@3.9` build dependency on macOS and Linux